File: app/src/main/java/com/another1dd/cybersportnews/model/ParseImg.java
package com.another1dd.cybersportnews.model;

import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.os.AsyncTask;
import android.util.Log;

import org.jsoup.Jsoup;
import org.jsoup.nodes.Document;
import org.jsoup.nodes.Element;

import java.io.BufferedInputStream;
import java.io.IOException;
import java.io.InputStream;
import java.net.URL;
import java.net.URLConnection;

import static android.content.ContentValues.TAG;


public class ParseImg extends AsyncTask<String, Void, Bitmap> {

    @Override
    protected Bitmap doInBackground(String... params) {

        String imageUrl = "";


        try {

            Document document = Jsoup.connect(params[0]).get();
            Element fearuteImg = document.select("").first();
            String temp = fearuteImg.getElementsByAttribute("style").toString();

            imageUrl = temp.substring(temp.indexOf("(") + 1, temp.indexOf(")"));


        } catch (IOException e) {
            e.printStackTrace();
        }

        return getImageBitmap(imageUrl);
    }

    private Bitmap getImageBitmap(String url) {
        Bitmap bm = null;
        try {

            // See what we are getting
            Log.i(TAG, "" + url);

            URL aURL = new URL(url);
            URLConnection conn = aURL.openConnection();
            conn.connect();

            InputStream is = conn.getInputStream();
            BufferedInputStream bis = new BufferedInputStream(is);
            bm = BitmapFactory.decodeStream(bis);

            bis.close();
            is.close();
        } catch (IOException e) {
            Log.e(TAG, "Error getting bitmap", e);
        }
        return bm;
    }
}

File: app/src/main/java/com/another1dd/cybersportnews/model/ParseImg.kt
package com.another1dd.cybersportnews.model

import android.content.ContentValues.TAG
import android.graphics.Bitmap
import android.graphics.BitmapFactory
import android.os.AsyncTask
import android.util.Log
import org.jsoup.Jsoup
import java.io.BufferedInputStream
import java.io.IOException
import java.net.URL


class ParseImg : AsyncTask<String, Void, Bitmap>() {
    override fun doInBackground(vararg params: String): Bitmap? {

        var imageUrl = ""


        try {

            val document = Jsoup.connect(params[0]).get()
            val fearuteImg = document.select("").first()
            val temp = fearuteImg.getElementsByAttribute("style").toString()

            imageUrl = temp.substring(temp.indexOf("(") + 1, temp.indexOf(")"))


        } catch (e: IOException) {
            e.printStackTrace()
        }

        return getImageBitmap(imageUrl)
    }

    private fun getImageBitmap(url: String): Bitmap? {
        var bm: Bitmap? = null
        try {

            // See what we are getting
            Log.i(TAG, "" + url)

            val aURL = URL(url)
            val conn = aURL.openConnection()
            conn.connect()

            val `is` = conn.getInputStream()
            val bis = BufferedInputStream(`is`)
            bm = BitmapFactory.decodeStream(bis)

            bis.close()
            `is`.close()
        } catch (e: IOException) {
            Log.e(TAG, "Error getting bitmap", e)
        }

        return bm
    }
}